app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt

```diff
@@ -86,14 +88,35 @@ class   SendReviewFragment : Fragment() {
         }
     }
 
+    fun createSendResultIntent(sendViewModel: SendViewModel, sendResult: String?): Intent {
+        val intent = Intent()
+        intent.putExtra("assetName", sendViewModel.getSelectedAsset().value!!.name)
+        intent.putExtra("amount", sendViewModel.getSelectedSendAmount().toDouble())
+        intent.putExtra("address", sendViewModel.getSelectedAddress().value!!)
+
+        if (sendViewModel.txID != null){
+            intent.putExtra("transactionId", sendViewModel.txID)
```

Comment:
Looks like this parameter is not set yet. Its value is updated after createSendResultIntent is called in 116 line
And it is actually equals to result

Reply:
No, Initialization of txID and sendResult variables is in SendViewModel class on transactions

There are 3 funcs and ways to do transactions in SendViewModel:
- sendOntAsset          - set up only sendResult
- sendNativeNeoAsset    - set up txID and sendResult with same value
- sendNeoTokenAsset     - set up only sendResult

i.e.

if (sendResult == null)
    100% failed
    
if (txID != null)
    100% success

if (txID == null)

    if (sendResult == valid transaction id)
        success
    else
        failed